packages/client/src/submodules/profile/Profile.ts

```diff
@@ -207,16 +207,11 @@ export class Profile {
     });
   }
 
-  async create(
+  async createWithHandle(
```

Comment:
is there any way to create a profile without a handle? then I would keep the method name simple and short `profile.create()`

Reply:
there is a claim profile - but that is separate

intention here was to keep the sdk as close to the GQL naming as possible

i think it makes sense to shorten here though, you're right